tests/compat.sh

```diff
@@ -422,6 +430,12 @@ add_common_ciphersuites()
     esac
 }
 
+# Ciphersuites usable only with Mbed TLS and OpenSSL
+# Each ciphersuite should appear two times, once with its OpenSSL name, once
+# with its Mbed TLS name.
+#
+# NOTE: for some reason RSA-PSK doesn't work with OpenSSL,
+# so RSA-PSK ciphersuites need to go in other sections.
```

Comment:
If you want, you can mention issue #1419 here, to tie everything together. It'll help maintainability.